src/lib.rs

```diff
@@ -21,6 +21,7 @@ extern crate rand_pcg;
 extern crate rayon;
 
 mod astar;
+mod dag_combine;
```

Comment:
Could you rename the module `digraph_combine` since it the input graph doesn't have to be a `dag`. The `dag_isomporphism` module was kind of a leftover from when `PyDiGraph` was called `PyDAG`